lib/content-services/src/lib/aspect-list/aspect-list.component.ts

```diff
@@ -57,13 +57,9 @@ export class AspectListComponent implements OnInit, OnDestroy {
     ngOnInit(): void {
         if (this.nodeId) {
             const node$ = this.nodeApiService.getNode(this.nodeId);
-            const customAspect$ = this.aspectListService.getCustomAspects()
-            .pipe(map(
-                (customAspects) => customAspects.flatMap((customAspect) => customAspect.entry.id)
-            ));
-            this.aspects$ = zip(node$, customAspect$).pipe(
-                tap(([node, customAspects]) => {
-                    this.nodeAspects = node.aspectNames.filter((aspect) => this.aspectListService.getVisibleAspects().includes(aspect) || customAspects.includes(aspect));
+            this.aspects$ = node$.pipe(
+                tap((node) => {
```

Comment:
can we add the type of the node? We should have such classes from what I remember